src/app/services/playlist.service.ts

```diff
@@ -4,49 +4,59 @@ import { SpotifyService } from "app/services/spotify.service";
 import { DomSanitizer } from "@angular/platform-browser";
 import { User } from "app/models/user";
 import { UserService } from "app/services/user.service";
+import { Room } from "app/models/room";
+import { AngularFireAuth } from "angularfire2/auth";
+import {
+  AngularFireDatabase,
+  FirebaseObjectObservable
+} from "angularfire2/database";
 
 @Injectable()
 export class PlaylistService {
```

Comment:
After this changes we are calling always data in the room object, I wonder if we should rename this service to something like RoomService... we can discuss about it, it is also strange to have a Playlist service with a room object as a member... it would make more sense to have a room or room service with a playlist object as member.